src/xcode/ENA/ENA/Source/Services/HealthCertificate/__tests__/HealthCertificateServiceTests.swift

```diff
@@ -373,33 +373,33 @@ class HealthCertificateServiceTests: CWATestCase {
 		}
 		
 		XCTAssertEqual(store.healthCertifiedPersons[safe: 0]?.healthCertificates.map { $0.base45 }, [thirdTestCertificate, secondVaccinationCertificate].map { $0.base45 })
-		XCTAssertEqual(service.healthCertifiedPersons[safe: 0]?.gradientType, .lightBlue)
+		XCTAssertEqual(service.healthCertifiedPersons[safe: 0]?.gradientType, .green)
 		XCTAssertEqual(try XCTUnwrap(store.healthCertifiedPersons[safe: 0]).unseenNewsCount, 0)
 		
 		XCTAssertEqual(store.healthCertifiedPersons[safe: 1]?.healthCertificates.map { $0.base45 }, [firstRecoveryCertificate.base45])
-		XCTAssertEqual(service.healthCertifiedPersons[safe: 1]?.gradientType, .solidGrey)
+		XCTAssertEqual(service.healthCertifiedPersons[safe: 1]?.gradientType, .green)
 		XCTAssertEqual(try XCTUnwrap(store.healthCertifiedPersons[safe: 1]).unseenNewsCount, 1)
 		
 		// Expired state does not increase unseen news count
 		XCTAssertEqual(service.unseenNewsCount.value, 3)
 		XCTAssertTrue(try XCTUnwrap(store.healthCertifiedPersons[safe: 1]?.healthCertificates[safe: 0]).isValidityStateNew)
 		
 		XCTAssertEqual(store.healthCertifiedPersons[safe: 2]?.healthCertificates.map { $0.base45 }, [firstVaccinationCertificate, firstTestCertificate, secondTestCertificate].map { $0.base45 })
-		XCTAssertEqual(service.healthCertifiedPersons[safe: 2]?.gradientType, .darkBlue)
+		XCTAssertEqual(service.healthCertifiedPersons[safe: 2]?.gradientType, .green)
 		XCTAssertEqual(try XCTUnwrap(store.healthCertifiedPersons[safe: 2]).unseenNewsCount, 2)
 		
 		// Set last person as preferred person and check that positions switched and gradients are correct
 		
 		service.healthCertifiedPersons.last?.isPreferredPerson = true
 		
 		XCTAssertEqual(store.healthCertifiedPersons[safe: 0]?.healthCertificates.map { $0.base45 }, [firstVaccinationCertificate, firstTestCertificate, secondTestCertificate].map { $0.base45 })
-		XCTAssertEqual(service.healthCertifiedPersons[safe: 0]?.gradientType, .lightBlue)
+		XCTAssertEqual(service.healthCertifiedPersons[safe: 0]?.gradientType, .green)
 		
 		XCTAssertEqual(store.healthCertifiedPersons[safe: 1]?.healthCertificates.map { $0.base45 }, [thirdTestCertificate, secondVaccinationCertificate].map { $0.base45 })
-		XCTAssertEqual(service.healthCertifiedPersons[safe: 1]?.gradientType, .mediumBlue)
+		XCTAssertEqual(service.healthCertifiedPersons[safe: 1]?.gradientType, .green)
 		
 		XCTAssertEqual(store.healthCertifiedPersons[safe: 2]?.healthCertificates.map { $0.base45 }, [firstRecoveryCertificate.base45])
-		XCTAssertEqual(service.healthCertifiedPersons[safe: 2]?.gradientType, .solidGrey)
+		XCTAssertEqual(service.healthCertifiedPersons[safe: 2]?.gradientType, .green)
```

Comment:
These all used to test that the colors are assigned and re-assigned in the correct order. Can you reinstate that somehow? And maybe have just one person that turns green when a certificate is added?